dev_guide/builds.adoc

```diff
@@ -1032,11 +1058,11 @@ $ oc set build-secret --source bc/sample-build sshsecret
 ----
 ====
 
-[[other-authentication]]
-===== Other
+[[gitconfig-authentication]]
```

Comment:
let's just call this section Git Configuration or something else not specific to auth, since you can use this mechanism to provide any git configuration you want.